results.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Charts Demo</title>
+  </head>
+  <body>
+    <canvas id="chart"></canvas>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0/dist/Chart.min.js"></script>
+    <script type="module" src="appp.js"></script>
```

Comment:
You don't need this script tag here since all of your logic for the results page is on `results.js`. This is what's giving you that error code you mentioned.